test/integration/parameter-types/01-install.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: kudo.dev/v1beta1
```

Comment:
Could you change the integration tests to use an explict operator like the `first-operator-test`? This would test the parameter parsing a well, this code here skips that completely as it directly creates the CRDs, circumventing the client side of KUDO.

Reply:
I added more parameters to the `cli-install-uninstall` E2E test. Happy to add an integration test as well, but I'd like to keep this test to make it easier to distinguish between issues that could occur in operator packages vs issues that could from the CRDs.